### PR TITLE
Improve Docker documentation, add a section on using Podman

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

File: content/en/docs/getting-started/docker/_index.md [hunk 1]
```diff
@@ -14,26 +14,85 @@ The docker image tag corresponding to this version of the documentation is:
 indigoiam/iam-login-service:{{< param version >}}
 {{% /pageinfo %}}
 
-### Configuration and run
+The IAM image has been tested to run properly with either Docker or Podman. The configuration for the
+IAM service container is the same but the details to run the container slightly differs. Differences
```
Comment (rmiccoli) [inline content/en/docs/getting-started/docker/_index.md]: differs -> differ

File: content/en/docs/getting-started/docker/_index.md [hunk 2]
```diff
@@ -14,26 +14,85 @@ The docker image tag corresponding to this version of the documentation is:
...
+This environment file will be passed to the container with the `--env-file` option of the 
+container engine. You can choose whatever file path/name you want.
+
+
+## Running the container
+
+### Docker
+
+First create a Docker network for the IAM service with the command (the example uses
+the network name `iam` but you are free to use anoter names as long as you use the same
```
Comment (rmiccoli) [inline content/en/docs/getting-started/docker/_index.md]: anoter names -> another name

File: content/en/docs/getting-started/docker/_index.md [hunk 3]
```diff
@@ -14,26 +14,85 @@ The docker image tag corresponding to this version of the documentation is:
...
+# Adapt the container name to the value you provided to --name in previous command
 $ docker logs -f iam-login-service
 ```
+
+### Podman
+
+When using Podman, the `docker` command must be replaced by `podman` with the following 
+differences in options:
+
+* `--restart` is silently ignored as start/stop of the container as boot time is controled
```
Comment (rmiccoli) [inline content/en/docs/getting-started/docker/_index.md]: controled -> controlled

File: content/en/docs/getting-started/docker/_index.md (hunk 1 above)
Comment (jouvin) [reply content/en/docs/getting-started/docker/_index.md]: Done

File: content/en/docs/getting-started/docker/_index.md (hunk 3 above)
Comment (jouvin) [reply content/en/docs/getting-started/docker/_index.md]: Done

File: content/en/docs/getting-started/docker/_index.md (hunk 2 above)
Comment (jouvin) [reply content/en/docs/getting-started/docker/_index.md]: Done

**Pull request merged**